src/peerly/services/peerlyIdentity.service.ts

```diff
@@ -145,4 +134,36 @@ export class PeerlyIdentityService extends PeerlyBaseConfig {
       this.handleApiError(error)
     }
   }
+
+  async approve10DLCBrand(
+    user: User,
+    { committeeName, peerlyIdentityId }: TcrCompliance,
+    campaignVerifyToken: string = '',
+  ) {
+    try {
+      const response: AxiosResponse<Approve10DLCBrandResponse> =
+        await lastValueFrom(
+          this.httpService.post(
+            `${this.baseUrl}/v2/tdlc/${peerlyIdentityId}/submit`,
+            {
+              campaign_verify_token: campaignVerifyToken,
+              entity_type: PEERLY_ENTITY_TYPE,
+              usecase: PEERLY_USECASE,
+              sample1: `Hello {first_name}, this is ${getUserFullName(user)} from ${committeeName}. Sample message content. Reply STOP to opt-out`,
+              sample2: `Hello {first_name}, this is ${getUserFullName(user)} from ${committeeName}. More sample message content. Reply STOP to opt-out`,
+            },
```

Comment:
is this `sample message` language intentional here as a placeholder?

Reply:
Yeah.. those are required by Peerly.. 

https://api-docs.peerly.com/reference/approve-10dlc-brand

<img width="1326" height="554" alt="image" src="https://github.com/user-attachments/assets/5d2ed5dd-9dfe-46aa-9c25-d5c6ec3fa19f" />